gitpod-io/gitpod
# [server] remove ssh functions scopes for workspace

## Description


[internal chat](https://gitpod.slack.com/archives/C01KGM9BH54/p1670144930725699)

## Related Issue(s)

Fixes #

## How to test

- Open workspace in preview env
- Check everything is OK - browser / desktop / ssh / gateway

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
